python/tank/context.py

```diff
@@ -1237,91 +1332,28 @@ def from_path(tk, path, previous_context=None):
 
     return Context(**context)
 
+
 ################################################################################################
 # serialization
 
 def serialize(context):
     """
     Serializes the context into a string.
 
-    Any Context object can be serialized to/deserialized from a string.
-    This can be useful if you need to pass a Context between different processes.
-    As an example, the ``tk-multi-launchapp`` uses this mechanism to pass the Context
-    from the launch process (e.g. for example Shotgun Desktop) to the
-    Application (e.g. Maya) being launched. Example:
-
-        >>> import sgtk
-        >>> tk = sgtk.sgtk_from_path("/studio.08/demo_project")
-        >>> ctx = tk.context_from_path("/studio.08/demo_project/sequences/AAA/ABC/Lighting/dirk.gently/work")
-        >>> context_str = sgtk.context.serialize(ctx)
-        >>> new_ctx = sgtk.context.deserialize(context_str)
-
-    .. info:: The currently authenticated user is also serialized together with the Context.
-
-    :param context: :class:`Context` instance to serialize
-    :returns: String representation
+    .. deprecated:: v0.18.2
```

Comment:
Finished criteria of this ticket mentions that this version number needs to be updated upon release.